sql/moz-fx-data-marketing-prod/acoustic/raw_recipient_v1/metadata.yaml

```diff
@@ -18,7 +18,7 @@ owners:
 
 scheduling:
   dag_name: bqetl_acoustic_raw_recipient_export
-  arguments: ["--submission_date", "{{ ds }}"]
```

Comment:
Is not super clear what the bug was about, but the change to use the partition parameter LGTM.

Reply:
It was resulting in wrong command generation: `FATAL Flags parsing error: Unknown command line flag 'submission_date'`